diesel_tests/tests/migrations.rs

```diff
@@ -0,0 +1,65 @@
+use diesel::types::{Bool, BigInt};
+use diesel::{select, LoadDsl};
+use diesel::result::QueryResult;
+use diesel::connection::Connection;
+use diesel::expression::dsl::sql;
+use diesel::migrations::revert_migration;
+use schema::*;
+
+
+// FIXME: Remove the duplicates of this function once expression level attributes
+// are stable (I believe this is in 1.7)
```

Comment:
We don't even need this here. Tests are only build for postgres or sqlite, not for both at the same time.